cadc-vos/build.gradle

```diff
@@ -18,14 +18,14 @@ group = 'org.opencadc'
 version = '1.1.5'
```

Comment:
Both the browser tests & the ability to update group read & write properties in vosui & cadc-vosui (user storage) was broken by an upgrade to cadc-util (1.2.8) where 'contentType' is required where it was not before. It percolated to user storage through the vospace client.

Reply:
The user storage vospace client usage was using the function associated with the first change I made - where a 'null' was passed in for the contentType. On speaking with Pat, it seemed to make sense to switch all the similar functions over to the new, fully supported constructor instead of the deprecated one, to reduce technical debt going forward.